### PR TITLE
update lesson java-mod-5 references

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +41,16 @@ the web.
...
-```txt
-https://github.com/learn-co-curriculum/phase-3-how-the-web-works-readme
+```text
+https://github.com/learn-co-curriculum/java-mod-5-how-the-web-works
 ```
 
 Turn into this:
 
-![Github Readme](https://curriculum-content.s3.amazonaws.com/phase-3/how-the-web-works-readme/github-readme.png)
+![Github Readme](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-servlets/github_screenshot.png)
```
Comment (kcleland0818) [inline README.md:50]: Is it possible to still show the URL bar in this screenshot?

Comment (kcleland0818) [inline README.md:50]: Is it possible to still show the URL bar in this screenshot?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+- Manipulate and test the structure of a request object.
+
+***
+
+## Key Vocab
+
+- **Web Framework**: software that is designed to support the development of
+  web applications. Web frameworks provide built-in tools for generating web
+  servers, turning Python objects into HTML, and more.
+- **Extension**: a package or module that adds functionality to a Flask
```
Comment (kcleland0818) [inline README.md]: Do we mean to have the word "Flask" in here?

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md:50]: yes, updated the image.

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: oops, thanks!

Comment (developerjaym) [reply README.md]: Relatedly, do you mean to have Python in the line above?

**Review by developerjaym — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: thanks, replaced with java

**Review by kcleland0818 — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**